styleguide/source/_patterns/02-molecules/article-stub~small-image.json

```diff
@@ -0,0 +1,21 @@
+{
+  "articleStub": {
+    "title_only": "",
+    "title": {
+      "title": "Link to something",
+      "href": "#",
+      "text": "Lorem ipsum dolor sit amet.",
+      "class": "ama__link-black",
```

Comment:
This should be `ama__link--black` since black is a modifier. Also I think black is the default so you may not need the class at all.